lux/config/config.exs

```diff
@@ -6,6 +6,10 @@ config :lux, :open_ai_models,
   default: "gpt-4o-mini",
   smartest: "gpt-4o"
 
+config :lux, :together_ai_models,
+  default: "mistralai/Mistral-7B-Instruct-v0.2",
```

Comment:
Do we really need to set these default models here for the developer? Perhaps the API has default models already? 
In tests we could set a value directly, without adding more global configs. Also, we better avoid labels like "smartest" as it they wouldn't age well

Reply:
Together ai doesn't provide any default model, user should select one before sending a request... but ok, I will delete this config then

Reply:
@rparcus but how will user select which model to use then? Like here https://github.com/Spectral-Finance/lux/pull/149/files#diff-f782925b9a7a05bd7d9f05e3d9a92ddc7bc888e85d702795246f221b6272296eR76 ?